week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,14 +7,22 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
-
+function convertToUSD(price) {
```

Comment:
Alternative approach
```suggestion
function convertToUSD(price) {
  return convert(price, 1.4);
}

function convertToBRL(price) {
  return convert(price, 4.91);
}

function convert(price, exchangeRate) {
  return price * exchangeRate * 0.99;
}
```